ios/MagnetScannerReact.swift

```diff
@@ -23,6 +23,10 @@ import Foundation
       .sendDeviceEventWithName("magnetscanner:itemfound", body: item)
     
     let url = item["url"] as! String
-    NotificationsHelper.notifyUser(url)
+    var channel: String? = nil
+    if item["channel"] != nil {
+      channel = item["channel"] as! String
+    }
+    NotificationsHelper.notifyUser(url, channel: channel)
   }
```

Comment:
Maybe we should be passing around a specific `MagnetScannerItem` instead of `Dictionary`